lib/escape.js

```diff
@@ -16,12 +25,9 @@ exports.escapeXMLText = function escapeXMLText (s) {
     .replace(/>/g, '&gt;')
 }
 
-exports.unescapeXML = function unescapeXML (s) {
+exports.unescapeXMLText = function escapeXMLText (s) {
   return s
     .replace(/\&(amp|#38);/g, '&')
     .replace(/\&(lt|#60);/g, '<')
     .replace(/\&(gt|#62);/g, '>')
-    .replace(/\&(quot|#34);/g, '"')
-    .replace(/\&(apos|#39);/g, "'")
-    .replace(/\&(nbsp|#160);/g, '\n')
```

Comment:
removed that last one not sure to understand why we have to unescape that 

@lloydwatkin ?